build.gradle

```diff
@@ -961,3 +965,18 @@ apply from: 'baseline.gradle'
 apply from: 'deploy.gradle'
 apply from: 'tasks.gradle'
 
+project(':iceberg-bom') {
+  apply plugin: 'java-platform'
+
+  dependencies {
+    constraints {
+      rootProject.allprojects.forEach {
+        if (it.name != 'iceberg-bom' && it != rootProject) {
+          add("api", project(it.path))
+        }
+      }
+    }
+  }
+
+  javaPlatform { allowDependencies() }
```

Comment:
can you elaborate why this is necessary?

Reply:
Hm - that (was) necessary in Nessie at least. Interesting that it's no longer necessary.